src/main/scala/org/broadinstitute/dsde/firecloud/service/WorkspaceService.scala

```diff
@@ -35,10 +36,21 @@ trait WorkspaceService extends HttpService with PerRequestCreator with FireCloud
     }
   }
 
+  def transformListWorkspaceRequest(entity: HttpEntity): HttpEntity = {
+    entity.as[List[RawlsWorkspaceResponse]] match {
+      case Right(lrwr) => lrwr.map(new UIWorkspaceResponse(_)).toJson.prettyPrint
+      case Left(error) =>
+        log.error("Unable to unmarshal entity -- " + error.toString)
+        entity
+    }
+  }
+
```

Comment:
Now that we have two methods that do roughly the same thing (this and transformSingleWorkspaceRequest) - it might be nice to refactor those into `FireCloudDirectiveUtils`